content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,240 @@
+
+To add a tax to a product, you can either set a tax in the :guilabel:`Customer Taxes` field of the
+**product template**, or use
+:doc:`fiscal positions <../../../finance/accounting/taxation/taxes/fiscal_positions>`.
+
+.. seealso::
+   - :doc:`../../../finance/accounting/taxation/taxes/taxes`
+   - :doc:`../../../finance/accounting/taxation/taxes/avatax`
+   - :doc:`../../../finance/accounting/taxation/taxes/taxcloud`
+   - :doc:`../../../finance/accounting/taxation/taxes/fiscal_positions`
+
+Tax display
+-----------
+
+Choosing the displayed price tax usually depends on a country's regulations or the type of customers
+**(B2B vs B2C)**. To select the type of price displayed, go to :menuselection:`Website -->
+Configuration --> Settings`, scroll down to the :guilabel:`Shop - Products` category, and select
+:guilabel:`Tax Excluded` or :guilabel:`Tax Included`.
+
+Note that this setting is **global**, and the tax-display type is the same for your website(s) and
+back-end. It is therefore not possible to select different tax displays for different websites. This
+may be a significant point of consideration when implementing a database with multiple ecommerce
+websites pertaining to different types of customers (i.e. B2B and B2C).
+
+- :guilabel:`Tax Excluded`: the price displayed on the website is **tax-excluded** and the tax is
+  computed at the cart-review step;
+- :guilabel:`Tax Included`: the price displayed on the website is **tax-included**.
+
+You can choose to display the type of pricing next to the product price, by going to
+:menuselection:`Website --> Site --> Homepage --> Shop`, selecting one of your product, and then
+:menuselection:`Edit --> Customize tab` and enabling :guilabel:`Tax Indication`.
+
+.. image:: price_management/price-tax-display-type.png
+   :align: center
+   :alt: Tax type displayed on the product page
+
+Price per unit
+==============
+
+It is possible to display a
+:doc:`price per unit <../../../inventory_and_mrp/inventory/management/products/uom>` on the product
+page. To do that, go to :menuselection:`Website --> Configuration --> Settings` and enable
+:guilabel:`Product Reference Price` under the :guilabel:`Shop - Products`. When enabled, make sure
+an amount is set in the :guilabel:`Base Unit Count` field of the **product template**, as well as in
+the :guilabel:`Sales Price`. The price per unit of measure can be found above the :guilabel:`Add to
+Cart` button on the product page.
+
+.. image:: price_management/price-cost-per-unit.png
+   :align: center
+   :alt: Cost per unit pricing on the product template
+
+.. image:: price_management/price-cost-per-unit-page.png
+   :align: center
+   :alt: Cost per unit pricing on the product page
+
+.. note::
+   Pay attention that having the price per unit may be **mandatory** in some countries.
+
+.. seealso::
+   - :doc:`../../../inventory_and_mrp/inventory/management/products/uom`
+
+Price configuration
+===================
+
+Pricelists
+----------
```

Comment:
Not sure that this heading is necessary. You could go for something like: _Price configuration: pricelists_